lib/fluent/plugin/in_kafka.rb

```diff
@@ -231,7 +235,16 @@ def consume
 
       messages.each { |msg|
         begin
-          es.add(Fluent::Engine.now, @parser.call(msg, @topic_entry))
+          record = @parser.call(msg, @topic_entry)
+          record_time = Fluent::Engine.now
```

Comment:
Don't create `Fluent::Engine.now` when option is true.
It allocates unnecessary object.